### PR TITLE
O blog

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -85,7 +90,9 @@ public void robotInit() {
      * and SmartDashboard integrated updating.
      */
     @Override
-    public void robotPeriodic() {
+    public void robotPeriodic()
```
Comment (msoucy) [inline src/main/java/frc/robot/Robot.java]: What's with the formatting here?

File: src/main/java/frc/robot/subsystems/ControlPanel.java [hunk 2]
```diff
@@ -6,14 +6,16 @@
...
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
+import io.github.oblarg.oblog.annotations.Log;
 
 public class ControlPanel extends SubsystemBase {
 
-    private SendableSpeedController spinnerMotor;
+    private final SendableSpeedController spinnerMotor;
 
+    @Log(tabName = "Configurable Values")
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/ControlPanel.java]: Should this be the `@Config` annotation?

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 3]
```diff
@@ -19,6 +20,7 @@
     private final SendableSpeedController intakeMotor;
     private final IDSolenoid deployPiston;
 
+    @Log(tabName = "Configurable Values")
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/Intake.java]: Should this be the `@Config` annotation?

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 4]
```diff
@@ -6,11 +6,13 @@
 import edu.wpi.first.wpilibj2.command.StartEndCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
+import io.github.oblarg.oblog.annotations.Log;
 
 public class Lift extends SubsystemBase {
 
     private SendableSpeedController elevatorMotor;
 
+    @Log(tabName = "Configurable Values")
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/Lift.java]: Should this be the `@Config` annotation?

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 5]
```diff
@@ -19,6 +20,10 @@
 
 public class Shooter extends SubsystemBase {
 
+    @Log(tabName = "Configurable Values")
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/Shooter.java]: Should this be the `@Config` annotation?

**Review by msoucy — CHANGES_REQUESTED** [review]

**Comment by msoucy** [comment]
If you have questions or need more info, [here](https://www.chiefdelphi.com/t/re-introducing-oblog-quick-easy-annotation-based-shuffleboard-api-for-java-teams/370533/) is the thread announcing oblog. You might be able to ask a question there (or have Ben or I do it)

**Comment by bot190** [comment]
We're going ahead with a simplified version of this PR.

**Pull request closed without merging**